Makefile

```diff
@@ -9,7 +9,7 @@ install:
 	$(GO) install ./...
 
 test:
-	$(GO) test --race ./...
+	$(GO) test --race ./pd-server ./pd-client ./server
```

Comment:
`$(GO) test $(shell $(GO) list ./... | grep -v /vendor/) -race` 

This can include future added package directly.